### PR TITLE
Add `night-not-found` feature

### PR DESCRIPTION
**Pull request by fregante**
light not found

### CODE REVIEW

**Comment by fregante** [comment]
Because why not

gif

File: source/features/night-not-found.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	filter: brightness(0.3) saturate(0.5);
+}
+
+[data-color-mode='dark'] .js-plaxify[height='49'] {
+	filter: brightness(0.9) saturate(0.5);
+}
+
+[data-color-mode='dark'] .js-plaxify[height='415'],
+[data-color-mode='dark'] .js-plaxify[height='75'],
+[data-color-mode='dark'] .js-plaxify[height='50'] {
```
Comment (yakov116) [inline source/features/night-not-found.css:30]: is?

**Comment by busches** [comment]
Can we up the contrast on the 404? I find it really hard to see.

**Comment by fregante** [comment]
Try seeing it in the dark.

Jokes aside, `filters` is rather limited and the 404/message block is a single piece, so to make that more visible I can only lighten the landscape, which kinda defeats the purpose. Maybe I'll try adjusting it when nighttime comes again.

**Comment by fregante** [comment]
No luck. Even a 0.05 brightness increase makes it lose that _night_ feeling. The "404" doesn't have great contrast but it's visible enough unless you use the dark theme in direct sunlight. There's text under it with perfect contrast anyway.

**Pull request merged**